pkg/portal/assets/main.js

```diff
@@ -0,0 +1,2 @@
",t),n.append(o)})),n.append(r("input[name='gorilla.csrf.Token']").first()),n.appendTo("body").submit()}}),o((function(){r.ajax({url:"/api/clusters",success:function(e){r.each(e,(function(e,t){r("#selResourceId").append(r("<option>").text(t))})),r("#selResourceId").selectpicker()},dataType:"json"}),r("#btnLogout").click((function(){r.redirect("/api/logout")})),r("#btnKubeconfig").click((function(){r.redirect(r("#selResourceId").val()+"/kubeconfig/new")})),r("#btnPrometheus").click((function(){window.location=r("#selResourceId").val()+"/prometheus"})),r("#btnSSH").click((function(){r.ajax({method:"POST",url:r("#selResourceId").val()+"/ssh/new",headers:{"X-CSRF-Token":r("input[name='gorilla.csrf.Token']").val()},contentType:"application/json",data:JSON.stringify({master:parseInt(r("#selMaster").val())}),success:function(e){if(e.error){var t=r("#tmplSSHAlertError").html();return(n=r(t)).find("span[data-copy='error']").text(e.error),void r("#divAlerts").html(n)}var n;t=r("#tmplSSHAlert").html(),(n=r(t)).find("span[data-copy='command'] > code").text(e.command),n.find("span[data-copy='command']").attr("data-copy",e.command),n.find("span[data-copy='password'] > code").text("********"),n.find("span[data-copy='password']").attr("data-copy",e.password),r("#divAlerts").html(n),r(".copy-button").click((function(){var e=r("<textarea class='style: hidden;' id='textarea'></textarea>");e.text(r(this).next().attr("data-copy")),e.appendTo("body"),(e=document.getElementById("textarea")).select(),e.setSelectionRange(0,e.value.length+1),document.execCommand("copy"),document.body.removeChild(e)}))},dataType:"json"})}))}))}},n={};function o(e){var r=n[e];if(void 0!==r)return r.exports;var a=n[e]={exports:{}};return t[e].call(a.exports,a,a.exports,o),a.exports}o.m=t,e=[],o.O=(t,n,r,a)=>{if(!n){var c=1/0;for(l=0;l<e.length;l++){for(var[n,r,a]=e[l],i=!0,d=0;d<n.length;d++)(!1&a||c>=a)&&Object.keys(o.O).every((e=>o.O[e](n[d])))?n.splice(d--,1):(i=!1,a<c&&(c=a));i&&(e.splice(l--,1),t=r())}return t}a=a||0;for(var l=e.length;l>0&&e[l-1][2]>a;l--)e[l]=e[l-1];e[l]=[n,r,a]},o.n=e=>{var t=e&&e.__esModule?()=>e.default:()=>e;return o.d(t,{a:t}),t},o.d=(e,t)=>{for(var n in t)o.o(t,n)&&!o.o(e,n)&&Object.defineProperty(e,n,{enumerable:!0,get:t[n]})},o.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||new Function("return this")()}catch(e){if("object"==typeof window)return window}}(),o.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),o.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},(()=>{var e={179:0};o.O.j=t=>0===e[t];var t=(t,n)=>{var r,a,[c,i,d]=n,l=0;for(r in i)o.o(i,r)&&(o.m[r]=i[r]);for(d&&d(o),t&&t(n);l<c.length;l++)a=c[l],o.o(e,a)&&e[a]&&e[a][0](),e[c[l]]=0;o.O()},n=self.webpackChunkaro_rp=self.webpackChunkaro_rp||[];n.forEach(t.bind(null,0)),n.push=t.bind(null,n.push.bind(n))})();var r=o.O(void 0,[216],(()=>o(873)));r=o.O(r)})();
```

Comment:
If we move this way I think it all should move out from golang path to `ui/` and have something like this this in golang: